crates/iota-sdk-ffi/src/types/graphql.rs

```diff
@@ -575,64 +575,21 @@ pub enum TransactionBlockKindInput {
     EndOfEpochTx,
 }
 
-#[derive(Clone, Debug, uniffi::Record)]
-pub struct BigInt {
-    pub value: String,
-}
-
-impl From<iota_graphql_client::query_types::BigInt> for BigInt {
-    fn from(value: iota_graphql_client::query_types::BigInt) -> Self {
-        BigInt { value: value.0 }
-    }
-}
-
-impl From<BigInt> for iota_graphql_client::query_types::BigInt {
-    fn from(value: BigInt) -> Self {
-        iota_graphql_client::query_types::BigInt(value.value)
-    }
-}
-
-#[derive(Clone, Debug, uniffi::Record)]
-pub struct DateTime {
-    pub value: String,
-}
-
-impl From<iota_graphql_client::query_types::DateTime> for DateTime {
-    fn from(value: iota_graphql_client::query_types::DateTime) -> Self {
-        DateTime { value: value.0 }
-    }
-}
-
-impl From<DateTime> for iota_graphql_client::query_types::DateTime {
-    fn from(value: DateTime) -> Self {
-        iota_graphql_client::query_types::DateTime(value.value)
-    }
-}
-
-/// Information about pagination in a connection.
 #[uniffi::remote(Record)]
 pub struct PageInfo {
-    /// When paginating backwards, are there more items?
     pub has_previous_page: bool,
-    /// Are there more items when paginating forwards?
     pub has_next_page: bool,
-    /// When paginating backwards, the cursor to continue.
     #[uniffi(default = None)]
     pub start_cursor: Option<String>,
-    /// When paginating forwards, the cursor to continue.
```

Comment:
Why remove the comments?

Reply:
I reverted the stuff in this PR for now. We'll have to go back and fix the rest

Reply:
https://github.com/iotaledger/iota-rust-sdk/issues/94